audio/yabridge/yabridge.SlackBuild

```diff
@@ -22,7 +22,7 @@
 #  OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 #  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-cd $(dirname $0) || exit 1 ; CWD=$(pwd)
+cd $(dirname $0) || exit 69 ; CWD=$(pwd)
```

Comment:
i don't think this is correct
if you want complete list, it's here: https://www.cyberciti.biz/faq/linux-bash-exit-status-set-exit-statusin-bash/
but i think simply exit 1 is just fine